assets/js/modules/analytics/datastore/report.js

```diff
@@ -122,6 +122,8 @@ const baseSelectors = {
 	 * @param {Object}         options                     Options for generating the report.
 	 * @param {string}         options.startDate           Required, unless dateRange is provided. Start date to query report data for as YYYY-mm-dd.
 	 * @param {string}         options.endDate             Required, unless dateRange is provided. End date to query report data for as YYYY-mm-dd.
+	 * @param {string}         [options.compareStartDate]  Optional. Start date to compare report data for as YYYY-mm-dd.
+	 * @param {string}         [options.compareEndDate]    Optional. End date to compare report data for as YYYY-mm-dd.
```

Comment:
Let's move these down to be after all required arguments rather than in between.

Reply:
Updated @aaemnnosttv